.syncpackrc

```diff
@@ -8,12 +8,13 @@
   ],
   "source": [
     "package.json",
+    "apps/*/package.json",
     "packages/*/package.json",
     "tooling/*/package.json"
```

Comment:
Just curious if this could be a glob `**/package.json`? Or would that be too permissive?

Reply:
We could certainly swap, yes. Although re-reading the docs I just realized we don't even need this value thanks to our `.pnpm-workspaces` file https://jamiemason.github.io/syncpack/config/source/